platform/security/sts/security-sts-propertyclaimshandler/src/test/java/org/codice/ddf/security/sts/claims/property/TestAttributeFileClaimsHandler.java

```diff
@@ -44,22 +48,24 @@
 
   @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
-  AttributeFileClaimsHandler attributeFileClaimsHandler = new AttributeFileClaimsHandler();
+  private final AttributeFileClaimsHandler attributeFileClaimsHandler =
+      new AttributeFileClaimsHandler();
 
-  ClaimCollection claimCollection;
+  private ClaimCollection claimCollection;
 
-  ClaimsParameters adminClaimsParameters;
+  private ClaimsParameters adminClaimsParameters;
 
-  ClaimsParameters localhostClaimsParameters;
+  private ClaimsParameters localhostClaimsParameters;
 
-  ClaimsParameters regexClaimsParameters;
+  private ClaimsParameters regexClaimsParameters;
 
-  ClaimsParameters unknownClaimsParameters;
+  private ClaimsParameters unknownClaimsParameters;
 
-  File userFile;
+  private File userFile;
 
   @Before
   public void setup() throws IOException {
+    System.setProperty(SystemBaseUrl.HOST, "admin");
```

Comment:
'admin' seems like an odd hostname. Just wanted to make sure that was intended.

Reply:
I updated it to `user1` instead.